pkg/chains/legacyevm/evm_txm.go

```diff
@@ -80,6 +86,16 @@ func newEvmTxm(
 	return
 }
 
+func validateTxWaitTimeForSubmitTransaction(cfg evmconfig.EVM) error {
+	if cfg.TxMinimumWaitTimeForConfirmation().Seconds() > time.Duration(600).Seconds() {
```

Comment:
Can we make `time.Duration(600).Seconds()` a constant

Reply:
Done